apps/endorsement-system/src/app/modules/endorsement/e2e/endorsement.spec.ts

```diff
@@ -154,6 +154,30 @@ describe('Endorsement', () => {
       statusCode: 405,
     })
   })
+  it(`POST /endorsement-list/:listId/endorsement/bulk should create a new endorsement and invalidate all endorsements`, async () => {
```

Comment:
Pedantry:
invalidate all other endorsements?

```suggestion
  it(`POST /endorsement-list/:listId/endorsement/bulk should create a new endorsement and invalidate all 
 other endorsements`, async () => {
```